tests/vue-package-tests/patches/App.vue

```diff
@@ -0,0 +1,14 @@
+<script setup lang="ts">
+import { AgCharts } from "ag-charts-vue3";
+
+// Insert options.partial here.
+</script>
+
+<template>
+  <main>
+    <AgCharts :options="options" />
+  </main>
```

Comment:
I think if you get rid of the `main`s, it'll fix the legend in the screenshots

Reply:
It didn't seem to help 🤷 Might be best to sit together on Monday and see if there is a simple fix for this, otherwise it's probably good enough right now to verify the package works.